core/src/main/java/org/opensearch/sql/analysis/model/CatalogName.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.sql.analysis.model;
+
+import java.util.List;
+import java.util.Set;
+import lombok.Getter;
+
+@Getter
+public class CatalogName {
+
+  public static final String DEFAULT_CATALOG_NAME = ".opensearch";
```

Comment:
why use ".opensearch" instead of "opensearch"?

Reply:
Since . is not allowed in a catalog name. `.opensearch` can never be a catalog name but `opensearch` could be a catalog name.